backend/Origam.Server/Controller/UIServiceController.cs

```diff
@@ -372,6 +388,17 @@ public IActionResult GetRow([FromBody]MasterRecordInput input)
         [HttpPost("[action]")]
         public IActionResult GetAggregations([FromBody]GetGroupsAggregations input)
         {
+            var sessionStore = sessionObjects.SessionManager.GetSession(
+                input.SessionFormIdentifier);
+            if (sessionStore is WorkQueueSessionStore workQueueSessionStore)
```

Comment:
Wouldn't it be right to encapsulate this into the SessionStore class so that the controller stays clean from these decisions?